library/src/blas1/rocblas_copy_strided_batched.cpp

```diff
@@ -85,15 +85,7 @@ namespace
                         "batch_count",
                         batch_count);
 
-        if(!x || !y)
-            return rocblas_status_invalid_pointer;
-
-        RETURN_ZERO_DEVICE_MEMORY_SIZE_IF_QUERIED(handle);
-
-        if(!incx || !incy || batch_count < 0)
-            return rocblas_status_invalid_size;
-
-        return rocblas_copy_template<NB, T>(
+        return rocblas_copy_template<NB, true, T>(
```

Comment:
BATCH_ADJUSTXY can be set to false for strided_batched to avoid overhead at kernel level.

Reply:
Oh, I thought all except non-batched required adjustments inside the kernel.

Reply:
It will technically still work outside of the kernel since the x and y pointers will be changed to point to the end of the first matrix and then the stride adjustments will continue to do so for every other matrix. I don't see any negative consequences from doing this, besides from being a little indirect. This is how I've done it for GER.

Reply:
Then go ahead and do it, after meeting this PR. The tests all passed (once I made the minor change to require `n > 0` before flagging `batch_count < 0` as an invalid argument).